.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,49 @@
+# 🔘 PR Type
+What kind of change does this PR introduce?
+
+<!-- Please check the one that applies to this PR using "x". -->
+
+- [ ] Bugfix
+- [ ] Enhancement
+- [ ] Code style update (formatting, local variables)
+- [ ] Refactoring (no functional changes, no api changes)
+- [ ] Build related changes
+- [ ] CI related changes
+- [ ] Documentation content changes
+- [ ] Other... Please describe:
+
+
+# 📜 Background
+Provide a brief explanation of why this pull request is needed. Include the problem you are solving or the functionality you are adding. Reference any related issues.
+
+Issue Link: #[issue_number]
+Context Link (if applicable):
+
+# 🔄 Changes
+Does this PR introduce a breaking change?
+
+- [ ] Yes, Incompatible API changes
+- [ ] No, Adds functionality (backwards compatible)
+- [ ] No, Bug fixes (backwards compatible)
+
+Changes:
+- What has been modified, added, or removed
+- Which are breaking changes
+
+Impact:
+- How it improves performance, fixes bugs, adds functionality, etc.
+- Impact on downstream apps, link to issues or PRs for migration
+- What to test
+
+# 🖼 Screenshot / 📹 Video
```

Comment:
I am not sure if Screenshots apply here

Reply:
merged already, but maybe you can make that part optional